### DISCUSSION TITLE
Komme/Gå til sfoweb.dk

### DISCUSSION THREAD

**Discussion by hmdrmch** [scaarup/aula#98] [Ideas]
Hej Scaarup,

Jeg vil høre om der er mulighed og stemning for at føje komme/gå tider fra sfoweb.dk til din integration?

Min datters skole benytter MeeBook til skoledagen, men når det bliver sfo tid, skal info komme fra sfoweb.dk.
Jeg har absolut intet indblik i kodningen og hvad der kræves og er nødvendigt for at det måske kunne lade sig gøre.

Lige pt kan man tilgå sfoweb via Aula app'en, som et link i toppen.
Det er også muligt gå direkte til siden i browser og logge ind med Tabulex, UNI-login eller NemLog-in.

Lad mig vide hvis det har interesse og du har brug for yderligere oplysninger.

**Comment by scaarup** [comment]
Hej @hmdrmch.
Når man tilgår sfoweb direkte fra Aula, så behøver man ikke logge ind i sfoweb?
Du kan jo evt. høre om der er flere, der er interesserede i denne feature. Så skulle det da nok kunne lade sig gøre.

**Reply by hmdrmch** [reply]
Ja hvis man åbner sfoweb inde fra Aula, så åbner den browser og logger direkte ind.
Man kan lige nå at skimte i adresselinjen at den redirecter fra borger.aula-login.tabulex.dk videre til saml.borger.tabulex.dk og så ind på sfoweb.dk til sidst.
